tests/test_examples.py

```diff
@@ -1186,6 +1191,333 @@ def test_example_get_current(self):
         )
 
 
+@pytest.mark.regression
+@pytest.mark.solve
+@pytest.mark.slow
+def test_regcoil_axisymmetric():
+    """Test axisymmetric regcoil solution."""
+    # make a simple axisymmetric vacuum equilibrium
+    eq = load("./tests/inputs/vacuum_circular_tokamak.h5")
+    # no phi_SV is needed since it is axisymmetric,
+    # so phi_mn should be zero when running REGCOIL
+    # especially with a nonzero alpha
```

Comment:
Is this alpha, the same as the alpha in signed distance or is it regularization alpha.
If it's the latter, can we call it something else?